EIPS/eip-4512.md

```diff
@@ -40,34 +37,122 @@ This standard adds the user role, and users can transfer their own usage rights.
 ### ERC-4512 Interface
 
 ```solidity
-event TransferUser(address from,address to,uint256 tokenId);
-event ApprovalUser(address indexed user, address indexed approved, uint256 indexed tokenId);
-function balanceOfUser(address user) external view returns (uint256 balance);
-function userOf(uint256 tokenId) external view returns (address user);  
-function safeTransferUserFrom(
-    address from,
-    address to,
-    uint256 tokenId
-) external;
-function safeTransferUserFrom(
-    address from,
-    address to,
-    uint256 tokenId,
-    bytes calldata data
-) external;
-function safeTransferAllFrom(
-     address from,
-     address to,
-     uint256 tokenId
-) external;    
-function safeTransferAllFrom(
-     address from,
-     address to,
-     uint256 tokenId,
-     bytes calldata data
-) external;
-function approveUser(address to, uint256 tokenId) external;
-function getApprovedUser(uint256 tokenId) external view returns (address operator);
+/**
+     * @dev Emitted when `tokenId` tokenUser is transferred from `from` to `to`.
+     */    
+    event TransferUser(address from,address to,uint256 tokenId);
+    
+    /**
+     * @dev Emitted when `user` enables `approved` to manage the `tokenId` tokenUser.
+     */ 
+    event ApprovalUser(address indexed user, address indexed approved, uint256 indexed tokenId);
+    
+    /**
+     * @dev Returns the number of usable token in ``user``'s account.
+     */    
+    function balanceOfUser(address user) external view returns (uint256 balance);
+    
+    /**
+     * @dev Returns the user of tokenId token
+     * Requirements:
+     *
+     * - `tokenId` must exist.
```

Comment:
i do believe you are able to instead do this for natspec
`@param tokenId <description>`

Reply:
Thanks again, it has been modified.